jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java

```diff
@@ -365,6 +365,11 @@ public long getContentRead()
         return _contentPosition;
     }
 
+    public long getHeaderLength()
```

Comment:
`_headerBytes` is an int and it make no sense to every have header size that needs a long.

Reply:
Yeah, I see that now.
Guess I just institutionally made it a long to match the `.getContentLength()` return value.
I'll change it.

Reply:
Changed.